.zshrc

```diff
@@ -18,6 +18,11 @@ plugins=(
 # Make all GIT repos into named directories.
 for repo in ~/git/*; do hash -d "$(basename "$repo")=$repo"; done
 
+# Make all workspaces into named directories.
+[ -d ~/development/workspace ] && for workspace in ~/development/workspace/*; do
```

Comment:
you can put your workspace anywhere with `dev volume workspace`

there is a command to retrieve the `workspace` location `dev workspace` which returns it as a string.